mods/bones/init.lua

```diff
@@ -211,81 +275,73 @@ minetest.register_on_dieplayer(function(player)
 		return
 	end
 
-	local player_inv = player:get_inventory()
-	if is_all_empty(player_inv) then
-		minetest.log("action", player_name .. " dies at " .. pos_string ..
-			". No bones placed")
-		if bones_position_message then
-			minetest.chat_send_player(player_name, S("@1 died at @2.", player_name, pos_string))
-		end
-		return
+	local callback=function(stk)
+		transfer_stack_to_bones(stk,current_dead_player)
 	end
 
-	-- check if it's possible to place bones, if not find space near player
-	if bones_mode == "bones" and not may_replace(pos, player) then
-		local air = minetest.find_node_near(pos, 1, {"air"})
-		if air and not minetest.is_protected(air, player_name) then
-			pos = air
-		else
-			bones_mode = "drop"
-		end
+	for i=1,#dead_player_callbacks do
+		local fun=dead_player_callbacks[i]
+		fun(player,callback)
 	end
 
-	if bones_mode == "drop" then
-		for _, list_name in ipairs(player_inventory_lists) do
-			for i = 1, player_inv:get_size(list_name) do
-				drop(pos, player_inv:get_stack(list_name, i))
+	local bones_conclusion
+
+	if not ( current_dead_player.bones_pos ) then
+		drop(current_dead_player.player_pos, ItemStack("bones:bones"))
+		if not ( current_dead_player.dropped ) then
+			bones_conclusion="No bones placed"
+			if bones_position_message then
+				minetest.chat_send_player(player_name, S("@1 died at @2.", player_name, pos_string))
+			end
+		else
+			bones_conclusion="Inventory dropped"
+			if bones_position_message then
+				minetest.chat_send_player(player_name, S("@1 died at @2, and dropped their inventory.", player_name, pos_string, public_conclusion))
```

Comment:
This won't work. Three arguments but only two `@` params in the translation string.

Reply:
>     * What happened to the logging?

https://github.com/imre84/minetest_game/blob/master/mods/bones/init.lua#L316
here's the logging. The code was restructured but I cannot remember any log entries being removed by me.